roles/deploy/defaults/main.yml

```diff
@@ -84,6 +84,8 @@ project_post_build_commands:
 # Post finalize commands are run with Ansible's `shell` module.
 # These are meant primarily for service restarts such as php5-fpm or memcached.
 project_post_finalize_commands:
+  - if wp core is-installed; then wp eval 'wp_clean_themes_cache();'; fi
+  - if wp core is-installed; then wp eval 'switch_theme(get_stylesheet());'; fi
```

Comment:
Smart, thanks! I'll merge into one command, but I'll keep the `if then` so it doesn't return `1`, which would cause playbook to fail/halt.